### PR TITLE
[FR] Add `Mappers' Guild`

### PR DESCRIPTION
**Pull request by Exanc**
French translation of the "Mappers' Guild" page for osu!wiki.

### CODE REVIEW

File: wiki/Mappers_Guild/fr.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Les récompenses sont basées sur le total de points d'un utilisateur. Les points disponibles d'un utilisateur sont une sous-section de leur total de points en suivant le même principe que les réserves de [kudosu](/wiki/Modding/Kudosu) "total gagné" et "disponible" d'osu!
+
+Les points sont gangnés pour différentes taches liées au mapping. Les gains varient entre les taches, bien qu'ils soient généralement pondérés en fonction du temps requis. Par exemple, mapper une difficulté `Hard` prend plus de temps que de mapper une dificulté `Easy`, la première rapportera donc plus de points que la deuxième.
+
+Les taches de création d'une difficilté rapportent plus de points à un utilisateur si la beatmap est associée à une quête. Finir une quête avant la date limite rapportera des points bonus pour chaque difficulté mappée et un bonus plat pour chaque utilisateur participant à la quête. Ce bonus plat<!-- J'ai pensée a utilisée "bonus fixe" mais je sais pas si ça évoque bien le comcept --> est perdue une foit la date limite dépassée, néanmoins, les bonus de difficultés sont gardé avec une certaine décroissance après des incréments de 0, 20 et 40 jours au-delà de la date limite.
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:45]:
```suggestion
Les tâches de création d'une difficulté rapportent plus de points à un utilisateur si la beatmap est associée à une quête. Finir une quête avant la date limite rapportera des points bonus pour chaque difficulté mappée et un bonus plat pour chaque utilisateur participant à la quête. Ce bonus fixe est perdue une fois la date limite dépassée, néanmoins, les bonus de difficultés sont gardé avec une certaine décroissance après des incréments de 0, 20 et 40 jours au-delà de la date limite.
```

File: wiki/Mappers_Guild/fr.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## L'Histoire
+
+La Mappers' Guild a été lancer avec un groupe de moins de 10 joueurs vers la fin de 2017. Compte tenu des informations qu'ont leurs avait fourni sur un nouvel artiste associé, [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19), ils étaient chargés de créer 5 beatmaps pour accompagner [la publication](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) sur l'artiste.
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:49]:
```suggestion
La Mappers' Guild a été lancer avec un groupe de moins de 10 joueurs vers fin 2017. Compte tenu des informations qu'ont leurs avait fourni sur un nouvel artiste associé, [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19), ils étaient chargés de créer 5 beatmaps pour accompagner [la publication](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) sur l'artiste.
```

File: wiki/Mappers_Guild/fr.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Pour promouvoir d'anciens artistes associés, le groupe recrutat de nouveaux membres et classa 8 beatmaps de différents artistes. Ces beatmaps furent publier dans un [poste](https://osu.ppy.sh/home/news/2018-03-17-new-mappers-guild-pack-release) accompagné d'un pack de beatmaps téléchargeable.<!-- TODO: beatmap pack article -->
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:51]:
```suggestion
Pour promouvoir d'anciens artistes associés, le groupe recrutait de nouveaux membres et classa 8 beatmaps de différents artistes. Ces beatmaps furent publier dans un [post](https://osu.ppy.sh/home/news/2018-03-17-new-mappers-guild-pack-release) accompagné d'un pack de beatmaps téléchargeable.
```

File: wiki/Mappers_Guild/fr.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+Dans la continuité de ces deux packs de beatmaps, la Mappers' Guild a continuellement agrandi son nombre de membres tout en publiant des packs pour les nouvelles annonces d'artistes associés, comme [High Tea Music](https://osu.ppy.sh/home/news/2018-08-13-new-featured-artist-high-tea-music), [Culprate](https://osu.ppy.sh/home/news/2018-11-21-new-featured-artist-culprate), ou encore [HyuN](https://osu.ppy.sh/beatmaps/artists/33), accompagné d'autres packs pour les musiques d'anciens artistes associés.
+
+### L'Automatisation
+
+Alors que le plan initial de la Mappers' Guild était d'avoir des éléments MMO tel que des quêtes, cela c'est avéré peut pratique à gérer tout en gérant manuellement les date limite pour le nombre limité de participants à la guilde. ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) et ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994) commencèrent donc le développement d'un site web pour automatiser la Mappers' Guild vers la fin de l'année 2018, permettant une gestion plus adéquate des récompenses et du system de quête/parti.
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:57]:
```suggestion
Alors que le plan initial de la Mappers' Guild était d'avoir des éléments MMO tel que des quêtes, cela c'est avéré peu pratique à gérer tout en gérant manuellement les date limite pour le nombre limité de participants à la guilde. ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) et ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994) commencèrent donc le développement d'un site web pour automatiser la Mappers' Guild vers la fin de l'année 2018, permettant une gestion plus adéquate des récompenses et du système de quête/parties.
```

File: wiki/Mappers_Guild/fr.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+Le [site web de la Mappers' Guild](https://mappersguild.com/) fut lancer le 1er Mars 2019 et c'est ici que tout ce qui concerne la Mappers' Guild est actuellement contrôlé. Depui sont ouvertures au public, la guilde a atteint un nombre de membres supérieurs à 500 et à publier des [articles de mise à jour](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) pour garder le reste de la communauté d'osu! à jour sur le contenu de mapping d'artistes associés.
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:59]:
```suggestion
Le [site web de la Mappers' Guild](https://mappersguild.com/) fut lancer le 1er Mars 2019 et c'est ici que tout ce qui concerne la Mappers' Guild est actuellement contrôlé. Depuis son ouverture au public, la guilde a atteint un nombre de membres supérieurs à 500 et à publier des [articles de mise à jour](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) pour garder le reste de la communauté d'osu! à jour sur le contenu de mapping d'artistes associés.
```

File: wiki/Mappers_Guild/fr.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+[flag_CL]: /wiki/shared/flag/CL.gif "Chile"
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:61]:
```suggestion
[flag_CL]: /wiki/shared/flag/CL.gif "Chili"
```

File: wiki/Mappers_Guild/fr.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+[flag_US]: /wiki/shared/flag/US.gif "United States"
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:62]:
```suggestion
[flag_US]: /wiki/shared/flag/US.gif "États-Unis"
```

File: wiki/Mappers_Guild/fr.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+
+Les parties sont des groupes de mappers qui s'inscrivent pour compléter une quête ensemble. Les mappers peuvent rejoindre autant de parties en même temps qu'il le souhaite.
+
+Les quêtes spéciales requièrent un plus haut rang dans la guilde pour y participer. Pour qu'une partie se qualifie dans un plus haut rang, au-moins la moitié des membres doivent être du rang requit ou d'un plus haut rang.
+
+### Les quêtes
+
+Les quêtes récompenses les mappers pour avoir suivi certaines exigences pendant la création de leurs maps d'artistes associés. Le plus souvent les quêtes impliquent qu'un certain nombre de beatmaps pour un artiste associé soit crée, cependant les quêtes impliquent aussi quelques petits gimmicks.
+
+Chaque quête à ses propres spécifications en therme de taille de partie, de rang, de prix d'acceptation et de date limite. Tous les membres d'une partie doivent payer le prix indiqué pour accepter la quête. Les prix sont souvent assez élevés pour les quêtes fantaisistes ou en solo.
```
Comment (Ganondorfosu) [inline wiki/Mappers_Guild/fr.md:31]:
```suggestion
Chaque quête a ses propres spécifications en termes de taille de groupe et de rang, de prix d'acceptation et de date limite. Tous les membres d'une partie doivent payer le prix indiqué pour accepter la quête. Les prix sont souvent assez élevés pour les quêtes fantaisistes ou en solo.
```

File: wiki/Mappers_Guild/fr.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+Les quêtes expirent apret 3 mois d'inactivité. N'importe quel utilisateur pour rouvrir une quête en dépensant leur réserve disponible de points de la Mappers' Guild.
```
Comment (Ganondorfosu) [inline wiki/Mappers_Guild/fr.md:33]:
```suggestion
Les quêtes expirent après 3 mois d'inactivité. N'importe quel utilisateur peut rouvrir une quête en dépensant sa réserve disponible de points de la Mappers' Guild.
```

File: wiki/Mappers_Guild/fr.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+
+La participation aux quêtes n'est pas requise pour les membres de la Mapper's Guild. Si un utilisateur le souhaite, il peut créer une beatmap seul pour obtenir des récompenses.
```
Comment (Ganondorfosu) [inline wiki/Mappers_Guild/fr.md:35]:
```suggestion
La participation aux quêtes n'est pas requise pour les membres de la Mappers' Guild. Si un utilisateur le souhaite, il peut créer une beatmap seul pour obtenir des récompenses.
```

File: wiki/Mappers_Guild/fr.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+### Les récompenses
+
+Les récompenses sont distribuées au travers d'un system de points. Quand un utilisateur atteint un seuil de points spécifier, il peut gagner un rang différent signifié par un badge de profil. Il y a trois niveaux de badges de profil: 100, 250 et 500 points respectivement.
+
+Les récompenses sont basées sur le total de points d'un utilisateur. Les points disponibles d'un utilisateur sont une sous-section de leur total de points en suivant le même principe que les réserves de [kudosu](/wiki/Modding/Kudosu) "total gagné" et "disponible" d'osu!
```
Comment (Ganondorfosu) [inline wiki/Mappers_Guild/fr.md:41]:
```suggestion
Les récompenses sont basées sur le total de points d'un utilisateur. Les points disponibles d'un utilisateur sont une sous-section de son total de points en suivant le même principe que les réserves de [kudosu](/wiki/Modding/Kudosu) "reçus au total" et "disponibles" d'osu!
```

File: wiki/Mappers_Guild/fr.md (hunk 2 above)
Comment (Ganondorfosu) [inline wiki/Mappers_Guild/fr.md:49]:
```suggestion
La Mappers' Guild a été lancée avec un groupe de moins de 10 joueurs vers fin 2017. Compte tenu des informations qui leur ont été fournies sur un nouvel artiste associé, [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19), ils étaient chargés de créer 5 beatmaps pour accompagner [la publication](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) sur l'artiste.
```

File: wiki/Mappers_Guild/fr.md (hunk 4 above)
Comment (Ganondorfosu) [inline wiki/Mappers_Guild/fr.md:57]:
```suggestion
Alors que le plan initial de la Mappers' Guild était d'avoir des éléments MMO tels que des quêtes, cela s'est avéré peu pratique à gérer tout en gérant manuellement les dates limites pour le nombre limité de participants à la guilde. ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) et ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994) commencèrent donc le développement d'un site web pour automatiser la Mappers' Guild vers la fin de l'année 2018, permettant une gestion plus adéquate des récompenses et du système de quêtes/groupes.
```

File: wiki/Mappers_Guild/fr.md (hunk 5 above)
Comment (Ganondorfosu) [inline wiki/Mappers_Guild/fr.md:59]:
```suggestion
Le [site web de la Mappers' Guild](https://mappersguild.com/) fut lancer le 1er Mars 2019 et c'est ici que tout ce qui concerne la Mappers' Guild est actuellement contrôlé. Depuis son ouverture au public, la guilde a atteint un nombre de membres supérieur à 500 et a publié des [articles de mise à jour](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) pour garder le reste de la communauté d'osu! à jour sur le contenu de mapping d'artistes associés.
```

**Review by Ganondorfosu — CHANGES_REQUESTED** [review]

File: wiki/Mappers_Guild/fr.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
+---
+tags:
+  - Mapper's Guild
+  - MG
+---
+
+# La Mappers' Guild
+
+La **Mappers' Guild** est un groupe de mappeurs responsables de la création des [beatmaps](/wiki/Beatmap) pour les [artistes associés](/wiki/Featured_Artists) d'osu! En utilisant un style de guilde MMO, elle permet aux mappeurs de gagner des récompenses en participant au procédé de classement des beatmaps des artistes associés et en coopérant dans des quêtes en lien avec le mapping.
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:9]:
```suggestion
La **Mappers' Guild** est un groupe de mappeurs responsables de la création des [beatmaps](/wiki/Beatmap) pour les [artistes associés](/wiki/Featured_Artists) d'osu!. En utilisant un style de guilde MMO, elle permet aux mappeurs de gagner des récompenses en participant au procédé de classement des beatmaps des artistes associés et en coopérant dans des quêtes en lien avec le mapping.
```

File: wiki/Mappers_Guild/fr.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+
+L'organisation et les discussions de la guilde peuvent être trouvées dans le salon `#mappers-guild` du [serveur Discord osu!dev](https://discord.gg/ppy).
+
+## Le système
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:13]:
```suggestion
## Fonctionnement
```

File: wiki/Mappers_Guild/fr.md [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+
+*Pour obtenir les informations les plus récentes sur le système utilisé par la Mappers' Guild, consultez la [section FAQ de son site web](https://mappersguild.com/faq).*
+
+Les mappeurs avec 3 maps classées ou plus peuvent rejoindre la guilde à tout moment via le [site web de la Mappers' Guild](https://mappersguild.com/) en authentifiant leur compte osu!. N'importe qui d'autre est libre d'observer les activités de mapping et les quêtes sur le site web.
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:17]:
```suggestion
Les mappeurs avec 3 maps classées ou plus peuvent rejoindre la guilde à tout moment via le [site web de la Mappers' Guild](https://mappersguild.com/) en authentifiant leur compte osu!. N'importe qui est libre d'observer les activités de mapping et les quêtes sur le site web.
```

File: wiki/Mappers_Guild/fr.md [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+
+Les mappeurs peuvent obtenir des récompenses en participant au procédé de classement des maps des artistes associés qui sont liés à la Mappers' Guild. Cela se concentre principalement sur la création de beatmaps, mais des récompenses sont aussi fournies aux utilisateurs qui ont contribué au [modding](/wiki/Modding), à la création de [storyboards](/wiki/Modding), ou encore en complétant des quêtes dans la guilde.
+
+### Les parties
+
+Les parties sont des groupes de mappeurs qui s'inscrivent pour compléter une quête ensemble. Les mappeurs peuvent rejoindre autant de parties en même temps qu'il le souhaite.
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:23]:
```suggestion
Les **parties** sont des groupes de mappeurs qui s'inscrivent pour compléter une quête ensemble. Les mappeurs peuvent rejoindre autant de parties en même temps qu'il le souhaite.
```

File: wiki/Mappers_Guild/fr.md [hunk 16]
```diff
@@ -0,0 +1,62 @@
...
+
+Les tâches de création d'une difficulté rapportent plus de points à un utilisateur si la beatmap est associée à une quête. Finir une quête avant la date limite rapportera des points bonus pour chaque difficulté mappée et un bonus plat pour chaque utilisateur participant à la quête. Ce bonus fixe est perdue une fois la date limite dépassée, néanmoins, les bonus de difficultés sont gardé avec une certaine décroissance après des incréments de 0, 20 et 40 jours au-delà de la date limite.
+
+## Histoire
+
+La Mappers' Guild a été lancer avec un groupe de moins de 10 joueurs vers fin 2017. Compte tenu des informations qu'ont leurs avait fourni sur un nouvel artiste associé, [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19), ils étaient chargés de créer 5 beatmaps pour accompagner [la publication](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) sur l'artiste.
```
Comment (Ganondorfosu) [inline wiki/Mappers_Guild/fr.md:49]:
```suggestion
La Mappers' Guild a été lancée avec un groupe de moins de 10 joueurs vers fin 2017. Compte tenu des informations qui leur ont été fournis sur un nouvel artiste associé, [MOtOLOiD](https://osu.ppy.sh/beatmaps/artists/19), ils étaient chargés de créer 5 beatmaps pour accompagner [la publication](https://osu.ppy.sh/home/news/2017-11-07-new-featured-artist-motoloid) sur l'artiste.
```

File: wiki/Mappers_Guild/fr.md [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
+
+### Les parties
+
+Les parties sont des groupes de mappeurs qui s'inscrivent pour compléter une quête ensemble. Les mappeurs peuvent rejoindre autant de parties en même temps qu'il le souhaite.
+
+Les quêtes spéciales requièrent un plus haut rang dans la guilde pour y participer. Pour qu'une partie se qualifie dans un plus haut rang, au-moins la moitié des membres doivent être du rang requit ou d'un plus haut rang.
+
+### Les quêtes
+
+Les quêtes récompensent les mappeurs pour avoir suivi certaines exigences pendant la création de leurs maps d'artistes associés. Le plus souvent, les quêtes impliquent qu'un certain nombre de beatmaps pour un artiste associé soit crée, cependant les quêtes impliquent aussi quelques petits gadgets.
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:29]:
```suggestion
Les **quêtes** récompensent les mappeurs pour avoir suivi certaines exigences pendant la création de leurs maps d'artistes associés. Le plus souvent, les quêtes impliquent qu'un certain nombre de beatmaps pour un artiste associé soit crée, cependant les quêtes impliquent aussi quelques petits gadgets.
```

File: wiki/Mappers_Guild/fr.md [hunk 18]
```diff
@@ -0,0 +1,62 @@
...
+
+## Le système
+
+*Pour obtenir les informations les plus récentes sur le système utilisé par la Mappers' Guild, consultez la [section FAQ de son site web](https://mappersguild.com/faq).*
+
+Les mappeurs avec 3 maps classées ou plus peuvent rejoindre la guilde à tout moment via le [site web de la Mappers' Guild](https://mappersguild.com/) en authentifiant leur compte osu!. N'importe qui d'autre est libre d'observer les activités de mapping et les quêtes sur le site web.
+
+Les mappeurs peuvent obtenir des récompenses en participant au procédé de classement des maps des artistes associés qui sont liés à la Mappers' Guild. Cela se concentre principalement sur la création de beatmaps, mais des récompenses sont aussi fournies aux utilisateurs qui ont contribué au [modding](/wiki/Modding), à la création de [storyboards](/wiki/Modding), ou encore en complétant des quêtes dans la guilde.
+
+### Les parties
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:21]:
```suggestion
### Parties
```

File: wiki/Mappers_Guild/fr.md [hunk 19]
```diff
@@ -0,0 +1,62 @@
...
+
+Les parties sont des groupes de mappeurs qui s'inscrivent pour compléter une quête ensemble. Les mappeurs peuvent rejoindre autant de parties en même temps qu'il le souhaite.
+
+Les quêtes spéciales requièrent un plus haut rang dans la guilde pour y participer. Pour qu'une partie se qualifie dans un plus haut rang, au-moins la moitié des membres doivent être du rang requit ou d'un plus haut rang.
+
+### Les quêtes
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:27]:
```suggestion
### Quêtes
```

File: wiki/Mappers_Guild/fr.md [hunk 20]
```diff
@@ -0,0 +1,62 @@
...
+
+Dans la continuité de ces deux packs de beatmaps, la Mappers' Guild a continuellement agrandi son nombre de membres tout en publiant des packs pour les nouvelles annonces d'artistes associés, comme [High Tea Music](https://osu.ppy.sh/home/news/2018-08-13-new-featured-artist-high-tea-music), [Culprate](https://osu.ppy.sh/home/news/2018-11-21-new-featured-artist-culprate), ou encore [HyuN](https://osu.ppy.sh/beatmaps/artists/33), accompagné d'autres packs pour les musiques d'anciens artistes associés.
+
+### L'Automatisation
+
+Alors que le plan initial de la Mappers' Guild était d'avoir des éléments MMO tel que des quêtes, cela c'est avéré peu pratique à gérer tout en gérant manuellement les date limite pour le nombre limité de participants à la guilde. ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) et ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994) commencèrent donc le développement d'un site web pour automatiser la Mappers' Guild vers la fin de l'année 2018, permettant une gestion plus adéquate des récompenses et du système de quête/parties.
```
Comment (Ganondorfosu) [inline wiki/Mappers_Guild/fr.md:57]:
```suggestion
Alors que le plan initial de la Mappers' Guild était d'avoir des éléments MMO tels que des quêtes, cela s'est avéré peu pratique à gérer tout en gérant manuellement les date limites pour le nombre limité de participants à la guilde. ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) et ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994) commencèrent donc le développement d'un site web pour automatiser la Mappers' Guild vers la fin de l'année 2018, permettant une gestion plus adéquate des récompenses et du système de quête/groupes.
```

File: wiki/Mappers_Guild/fr.md [hunk 21]
```diff
@@ -0,0 +1,62 @@
...
+
+Le [site web de la Mappers' Guild](https://mappersguild.com/) fut lancer le 1er Mars 2019 et c'est ici que tout ce qui concerne la Mappers' Guild est actuellement contrôlé. Depuis son ouverture au public, la guilde a atteint un nombre de membres supérieurs à 500 et à publier des [articles de mise à jour](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) pour garder le reste de la communauté d'osu! à jour sur le contenu de mapping d'artistes associés.
```
Comment (Ganondorfosu) [inline wiki/Mappers_Guild/fr.md:59]:
```suggestion
Le [site web de la Mappers' Guild](https://mappersguild.com/) fut lancer le 1er Mars 2019 et c'est ici que tout ce qui concerne la Mappers' Guild est actuellement contrôlé. Depuis son ouverture au public, la guilde a atteint un nombre de membres supérieur à 500 et a publié des [articles de mise à jour](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) pour garder le reste de la communauté d'osu! à jour sur le contenu de mapping d'artistes associés.
```

**Review by Ganondorfosu — CHANGES_REQUESTED** [review]

File: wiki/Mappers_Guild/fr.md [hunk 22]
```diff
@@ -0,0 +1,62 @@
...
+
+Les quêtes récompensent les mappeurs pour avoir suivi certaines exigences pendant la création de leurs maps d'artistes associés. Le plus souvent, les quêtes impliquent qu'un certain nombre de beatmaps pour un artiste associé soit crée, cependant les quêtes impliquent aussi quelques petits gadgets.
+
+Chaque quête a ses propres spécifications en termes de taille de groupe et de rang, de prix d'acceptation et de date limite. Tous les membres d'une partie doivent payer le prix indiqué pour accepter la quête. Les prix sont souvent assez élevés pour les quêtes fantaisistes ou en solo.
+
+Les quêtes expirent après 3 mois d'inactivité. N'importe quel utilisateur peut rouvrir une quête en dépensant sa réserve disponible de points de la Mappers' Guild.
+
+La participation aux quêtes n'est pas requise pour les membres de la Mappers' Guild. Si un utilisateur le souhaite, il peut créer une beatmap seul pour obtenir des récompenses.
+
+### Les récompenses
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:37]:
```suggestion
### Récompenses
```

File: wiki/Mappers_Guild/fr.md [hunk 23]
```diff
@@ -0,0 +1,62 @@
...
+
+Les récompenses sont distribuées au travers d'un système de points. Quand un utilisateur atteint un seuil de points spécifié, il peut gagner un rang différent signifié par un badge de profil. Il y a trois niveaux de badges de profil : 100, 250 et 500 points respectivement.
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:39]:
```suggestion
Les **récompenses** sont distribuées au travers d'un système de points. Quand un utilisateur atteint un seuil de points spécifié, il peut gagner un rang différent signifié par un badge de profil. Il y a trois niveaux de badges de profil : 100, 250 et 500 points respectivement.
```

File: wiki/Mappers_Guild/fr.md [hunk 24]
```diff
@@ -0,0 +1,62 @@
...
+
+Les récompenses sont basées sur le total de points d'un utilisateur. Les points disponibles d'un utilisateur sont une sous-section de son total de points en suivant le même principe que les réserves de [kudosu](/wiki/Modding/Kudosu) "reçus au total" et "disponibles" d'osu!
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:41]:
```suggestion
Les récompenses sont basées sur le total de points d'un utilisateur. Les points disponibles d'un utilisateur sont une sous-section de son total de points en suivant le même principe que les [kudosu](/wiki/Modding/Kudosu) "reçus au total" et "disponibles" d'osu!
```

File: wiki/Mappers_Guild/fr.md [hunk 25]
```diff
@@ -0,0 +1,62 @@
...
+
+Pour promouvoir d'anciens artistes associés, le groupe recrutait de nouveaux membres et classa 8 beatmaps de différents artistes. Ces beatmaps furent publier dans un [post](https://osu.ppy.sh/home/news/2018-03-17-new-mappers-guild-pack-release) accompagné d'un pack de beatmaps téléchargeable.
+
+Dans la continuité de ces deux packs de beatmaps, la Mappers' Guild a continuellement agrandi son nombre de membres tout en publiant des packs pour les nouvelles annonces d'artistes associés, comme [High Tea Music](https://osu.ppy.sh/home/news/2018-08-13-new-featured-artist-high-tea-music), [Culprate](https://osu.ppy.sh/home/news/2018-11-21-new-featured-artist-culprate), ou encore [HyuN](https://osu.ppy.sh/beatmaps/artists/33), accompagné d'autres packs pour les musiques d'anciens artistes associés.
+
+### L'Automatisation
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:55]:
```suggestion
### Automatisation
```

File: wiki/Mappers_Guild/fr.md [hunk 26]
```diff
@@ -0,0 +1,62 @@
...
+
+Alors que le plan initial de la Mappers' Guild était d'avoir des éléments MMO tels que des quêtes, cela s'est avéré peu pratique à gérer tout en gérant manuellement les date limites pour le nombre limité de participants à la guilde. ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) et ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994) commencèrent donc le développement d'un site web pour automatiser la Mappers' Guild vers la fin de l'année 2018, permettant une gestion plus adéquate des récompenses et du système de quête/groupes.
+
+Le [site web de la Mappers' Guild](https://mappersguild.com/) fut lancer le 1er Mars 2019 et c'est ici que tout ce qui concerne la Mappers' Guild est actuellement contrôlé. Depuis son ouverture au public, la guilde a atteint un nombre de membres supérieur à 500 et a publié des [articles de mise à jour](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) pour garder le reste de la communauté d'osu! à jour sur le contenu de mapping d'artistes associés.
```
Comment (DoryanR) [inline wiki/Mappers_Guild/fr.md:59]:
```suggestion
Le [site web de la Mappers' Guild](https://mappersguild.com/) fut lancer le 1er Mars 2019 et c'est ici que tout ce qui concerne la Mappers' Guild est actuellement contrôlé. Depuis son ouverture au public, la guilde a atteint un nombre de membres supérieur à 500 et a publié des [articles de mise à jour](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) pour garder le reste de la communauté d'osu! à jour sur le contenu de mapping des artistes associés.
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

**Review by DoryanR — APPROVED** [review]

**Review by DoryanR — APPROVED** [review]
Can be merge 👍

**Review by DoryanR — APPROVED** [review]
Can be merge

**Review by TicClick — APPROVED** [review]
merging under the assumption that the three unresolved comments were addressed in one way or another

**Pull request merged**